src/Indicator.vala

```diff
@@ -32,6 +32,12 @@ public class Sound.Indicator : Wingpanel.Indicator {
     private Notify.Notification? notification;
     private Services.VolumeControlPulse volume_control;
 
+    private ShellKeyGrabber? key_grabber = null;
+    private ulong key_grabber_id = 0;
+    private GLib.GenericSet<uint> volume_up_action_ids = new GLib.GenericSet<uint> (null, null);
+    private GLib.GenericSet<uint> volume_down_action_ids = new GLib.GenericSet<uint> (null, null);
+    private GLib.GenericSet<uint> volume_mute_action_ids = new GLib.GenericSet<uint> (null, null);
+
```

Comment:
You could use a Gee.MultiMap, so there's less code to update here later if we decide to handle more keybindings.

Reply:
Done